qemu/tests/cfg/boot_nbdimage_with_qsd.cfg

```diff
@@ -0,0 +1,41 @@
+- boot_nbdimage_with_qsd:
+    virt_test_type = qemu
+    type = boot_nbdimage_with_qsd
+    required_qemu = [6.0, )
+    start_vm = no
+    kill_vm = yes
+    # prepare an installed os image via converting
+    convert_source = ${images}
+    convert_target = stg1
+    image_name_stg1 = "images/stg1"
+    image_format_stg1 = raw
+    enable_nbd_stg1 = no
+    storage_type_stg1 = filesystem
+    remove_image_stg1 = 'yes'
```

Comment:
No need to use a nbd storage backend for this test, because we just setup an internal nbd server with qsd, everything is configurable, and there is no remote nbd server.
Then we can remove these 3 lines

note remove_image takes effect only when stg1 is in images